apps/site-app/src/containers/Accounts/Transfer/TransferWidget/index.tsx

```diff
@@ -23,20 +23,24 @@ type VisualProps={
   rate: number,
 
   onAccountValue: ChangeCB,
+  resetToDefault: number;
   forceValidate: boolean,
   onSubmit: StrictButtonProps["onClick"],
   button: MessageDescriptor,
-  destinationAddress: string,
+  destinationAddress: MessageDescriptor,
```

Comment:
The typing for UxInput let through basically anything, and so hid a lot of errors like this one.  Passing through a string where the code expected a MessageDescriptor caused all sorts of problems in our pages.